keyboards/yushakobo/ergo68/info.json

```
@@ -0,0 +1,188 @@
+{
+    "manufacturer": "yushakobo",
+    "keyboard_name": "ergo68",
+    "maintainer": "yushakobo",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "rgb_matrix": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "debounce": 5,
+    "processor": "atmega32u4",
```

1. Pro micro mentioned in readme.md and confirmed in build guide
2. `5` is default value for debounce
3. removed settings in `features` that are disabled/false by default
```suggestion
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "rgb_matrix": true,
    },
    "development_board": "promicro",
    "diode_direction": "COL2ROW",
    "matrix_pins": {
        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2"],
        "rows": ["D4", "C6", "D7", "E6", "B4"]
    },
```